pkg/util/arangod/cleanout_server.go

```diff
@@ -0,0 +1,99 @@
+//
+// DISCLAIMER
+//
+// Copyright 2018 ArangoDB GmbH, Cologne, Germany
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Copyright holder is ArangoDB GmbH, Cologne, Germany
+//
+// Author Ewout Prangsma
+//
+
+package arangod
+
+import (
+	"context"
+	"fmt"
+
+	driver "github.com/arangodb/go-driver"
+	"github.com/arangodb/go-driver/agency"
+)
+
+// CleanoutJobStatus is a strongly typed status of an agency cleanout-server-job.
+type CleanoutJobStatus struct {
+	state  string
+	reason string
+}
+
+// IsFailed returns true when the job is failed
+func (s CleanoutJobStatus) IsFailed() bool {
+	return s.state == "Failed"
+}
+
+// IsDone returns true when the job is finished
+func (s CleanoutJobStatus) IsDone() bool {
```

Comment:
Should be called `Finished` here as well, see below.

Reply:
Changed Done to Finished & `IsDone` to `IsFinished` (which is not being used)